tutteli-gradle-kotlin-module-info/src/main/groovy/ch/tutteli/gradle/kotlin/module/info/ModuleInfoPlugin.groovy

```diff
@@ -37,6 +37,8 @@ class ModuleInfoPlugin implements Plugin<Project> {
             project.compileModuleJava.configure {
                 dependsOn project.compileKotlin
                 destinationDir = project.compileKotlin.destinationDir
+                sourceCompatibility = 9
```

Comment:
I have in mind that this causes problems for JDK11, but maybe I am wrong

Reply:
What kind of problems? I compiled with JDK 12 it worked (see below)

Reply:
Some gradle failure similar to the one we want to fix with this PR but IMO your test shows that it works, so we are good.